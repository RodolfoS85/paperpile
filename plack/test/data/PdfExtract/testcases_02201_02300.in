---
file: ../../../PDFREPOSITORY/002201.pdf
---
file: ../../../PDFREPOSITORY/002202.pdf
---
file: ../../../PDFREPOSITORY/002203.pdf
---
file: ../../../PDFREPOSITORY/002204.pdf
---
file: ../../../PDFREPOSITORY/002205.pdf
---
file: ../../../PDFREPOSITORY/002206.pdf
---
file: ../../../PDFREPOSITORY/002207.pdf
---
file: ../../../PDFREPOSITORY/002208.pdf
---
file: ../../../PDFREPOSITORY/002209.pdf
---
file: ../../../PDFREPOSITORY/002210.pdf
---
file: ../../../PDFREPOSITORY/002211.pdf
---
file: ../../../PDFREPOSITORY/002212.pdf
---
file: ../../../PDFREPOSITORY/002213.pdf
---
file: ../../../PDFREPOSITORY/002214.pdf
---
file: ../../../PDFREPOSITORY/002215.pdf
---
file: ../../../PDFREPOSITORY/002216.pdf
---
file: ../../../PDFREPOSITORY/002217.pdf
---
file: ../../../PDFREPOSITORY/002218.pdf
---
file: ../../../PDFREPOSITORY/002219.pdf
---
file: ../../../PDFREPOSITORY/002220.pdf
---
file: ../../../PDFREPOSITORY/002221.pdf
---
file: ../../../PDFREPOSITORY/002222.pdf
---
file: ../../../PDFREPOSITORY/002223.pdf
---
file: ../../../PDFREPOSITORY/002224.pdf
---
file: ../../../PDFREPOSITORY/002225.pdf
