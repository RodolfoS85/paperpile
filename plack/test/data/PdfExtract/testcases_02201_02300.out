---
authors: 'Ziv-Ukelson, Michal and Gat-Viks, Irit and Wexler, Ydo'
title: A faster algorithm for RNA co-folding
---
authors: 'Guha, Sudipto and Rastogi, Rajeev and Shim, Kyuseok'
title: 'ROCK: A ROBUST CLUSTERING ALGORITHM FOR CATEGORICAL ATTRIBUTES'
---
authors: 'Heckerman, David'
title: A Tutorial on Learning With Bayesian Networks
---
authors: 'Win, Joe and Morgan, William and Bos, Jorunn and Krasileva, Ksenia V. and Cano, Liliana M. and Chaparro-Garcia, Angela and Ammar, Randa and Staskawicz, Brian J. and Kamoun, Sophien'
doi: 10.1105/tpc.107.051037
title: Adaptive Evolution Has Targeted the C-Terminal Domain of the RXLR Effectors of Plant Pathogenic Oomycetes
---
authors: 'Galagan, James E. and Calvo, Sarah E. and Borkovich, Katherine A. and Selker, Eric U. and Read, Nick D. and Jaffe, David and Hugh, William Fitz and Ma, Li-Jun and Smirnov, Serge and Purcell, Seth and Rehman, Bushra and Elkins, Timothy and Engels, Reinhard and Wang, Shunguang and Nielsen, Cydney B. and Butler, Jonathan and Endrizzi, Matthew and Qui, Dayong and Ianakiev, Peter and Bell-Pedersen, Deborah and Nelson, Mary Anne and Werner-Washburne, Margaret and Selitrennikoff, Claude P. and Kinsey, John A. and Braun, Edward L. and Zelter, Alex and Schulte, Ulrich and Kothe, Gregory O. and Jedd, Gregory and Mewes, Werner and Staben, Chuck and Marcotte, Edward and Greenberg, David and Roy, Alice and Foley, Karen and Naylor, Jerome and Stange-Thomann, Nicole and Barrett, Robert and Gnerre, Sante and Kamal, Michael and Kamvysselis, Manolis and Mauceli, Evan and Bielke, Cord and Rudd, Stephen and Frishman, Dmitrij and Krystofova, Svetlana and Rasmussen, Carolyn and Metzenberg, Robert L. and Perkins, David D. and Kroken, Scott and Cogoni, Carlo and Macino, Giuseppe and Catcheside, David and Li, Weixi and Pratt, Robert J. and Osmani, Stephen A. and De Souza, Colin P. C. and Glass, Louise and Orbach, Marc J. and Berglund, J. Andrew and Voelker, Rodger and Yarden, Oded and Plamann, Michael and Seiler, Stephan and Dunlap, Jay and Radford, Alan and Aramayo, Rodolfo and Natvig, Donald O. and Alex, Lisa A. and Mannhaupt, Gertrud and Ebbole, Daniel J. and Freitag, Michael and Paulsen, Ian and Sachs, Matthew S. and Lander, Eric S. and Nusbaum, Chad and Birren, Bruce'
title: The genome sequence of the filamentous fungus Neurospora crassa
---
authors: 'Harrison, Rhonda and De Lisi, Charles'
title: Condition specific transcription factor binding site characterization in Saccharomyces cerevisiae
---
authors: 'Skaletsky, Helen and Kuroda-Kawaguchi, Tomoko and Minx, Patrick J. and Cordum, Holland S. and Hillier, LaDeana and Brown, Laura G. and Repping, Sjoerd and Pyntikova, Tatyana and Ali, Johar and Bieri, Tamberlyn and Chinwalla, Asif and Delehaunty, Andrew and Delehaunty, Kim and Du, Hui and Fewell, Ginger and Fulton, Lucinda and Fulton, Robert and Graves, Tina and Hou, Shun-Fang and Latrielle, Philip and Leonard, Shawn and Mardis, Elaine and Maupin, Rachel and McPherson, John and Miner, Tracie and Nash, William and Nguyen, Christine and Ozersky, Philip and Pepin, Kymberlie and Rock, Susan and Rohlfing, Tracy and Scott, Kelsi and Schultz, Brian and Strong, Cindy and Tin-Wollam, Aye and Yang, Shiaw-Pyng and Waterston, Robert H. and Wilson, Richard K. and Rozen, Steve and Page, David C.'
title: The male-specific region of the human Y chromosome is a mosaic of discrete sequence classes
---
authors: 'Xiao, Bin and Li, Wei and Guo, Gang and Li, Bo Sheng and Liu, Zhen and Jia, Ke Ran and Guo, Ying and Mao, Xu Hu and Zou, Quan Ming'
doi: 10.1007/s00284-008-9318-2
title: Identification of Small Noncoding RNAs in Helicobacter pylori by a Bioinformatics-Based Approach
---
authors: 'Minton, C. D. T.'
title: Catching Lapwings with cannon nets
---
authors: 'Konigson, Sara and Hemmingsson, Malin and Lunneryd, SVEN-GUNNAR and Lundstrom, Karl'
doi: 10.1080/17451000601072596
title: 'Seals and fyke nets: An investigation of the problem and its possible solution'
---
authors: 'Demr, Semra'
title: Influence of Arbuscular Mycorrhiza on Some Physiological Growth Parameters of Pepper
---
authors: 'Bushnell, David A. and Westover, Kenneth D. and Davis, Ralph E. and Kornberg, Roger D.'
title: 'Structural Basis of Transcription: An RNA Polymerase II-TFIIB Cocrystal at 4.5 Angstroms'
---
authors: 'Lu, Shanfa and Sun, Ying-Hsuan and Shi, Rui and Clark, Catherine and Li, Laigeng and Chiang, Vincent L.'
doi: 10.1105/tpc.105.033456
title: Novel and Mechanical Stress-Responsive MicroRNAs in Populus trichocarpa That Are Absent from Arabidopsis
---
authors: 'Peschon, Jacques J. and Slack, Jennifer L. and Reddy, Pranitha and Stocking, Kim L. and Sunnarborg, Susan W. and Lee, David C. and Russell, William E. and Castner, Beverly J. and Johnson, Richard S. and Fitzner, Jeffrey N. and Boyce, Rogely W. and Nelson, Nicole and Kozlosky, Carl J. and Wolfson, Martin F. and Rauch, Charles T. and Cerretti, Douglas Pat and Paxton, Raymond J. and March, Carl J.'
title: An Essential Role for Ectodomain Shedding in Mammalian Development
---
authors: 'Scherf, Matthias and Klingenhoff, Andreas and Frech, Kornelie and Quandt, Kerstin and Schneider, Ralf and Grote, Korbinian and Frisch, Matthias and Gailus-Durner, Valerie and Seidel, Alexander and Brack-Werner, Ruth and Werner, Thomas'
doi: 10.1101/gr.154601
title: First Pass Annotation of Promoters on Human Chromosome 22
---
authors: 'Spellman, Paul T. and Sherlock, Gavin and Zhang, Michael Q. and Iyer, Vishwanath R. and Anders, Kirk and Eisen, Michael B. and Brown, Patrick O. and Botstein, David and Futcher, Bruce'
title: Comprehensive Identification of Cell Cycle-regulated Genes of the Yeast Saccharomyces cerevisiae by Microarray Hybridization
---
authors: 'Aggarwal, Alok and Klawe, Maria M. and Moran, Shlomo and Shor, Peter and WIlber, Robert'
title: Geometric Applications of A Matrix Se~ching Algorithm
---
authors: 'Davuluri, Ramana V. and Suzuki, Yutaka and Sugano, Sumio and Zhang, Michael Q.'
doi: 10.1101/gr.146000
title: CART Classification of Human 5Ј UTR Sequences
---
authors: 'Dietrich, Rosemary C. and Peris, Marian J. and Seyboldt, Andrew S. and Padgett, Richard A.'
doi: 10.1128/MCB.21.6.1942-1952.2001
title: Role of the 3Ј Splice Site in U12-Dependent Intron Splicing
---
authors: 'Kupiec, Julian'
title: An Algo ri thm for Estimating the Parameters of Unrestricted I-Iidden Stochastic Context-Free Grammars
---
authors: 'Mccullough, Andrew J. and Berget, Susan M.'
title: An Intronic Splicing Enhancer Binds U1 snRNPs To Enhance Splicing and Select 5Ј Splice Sites
---
authors: 'Merkle, Ingrid and van Ooij, Mark J. M. and van Kuppeveld, Frank J. M. and Glaudemans, Dirk H. R. F. and Galama, Jochem M. D. and Henke, Andreas and Zell, Roland and Melchers, Willem J. G.'
doi: 10.1128/JVI.76.19.9900-9909.2002
title: Biological Significance of a Human Enterovirus B-Specific RNA Element in the 3Ј Nontranslated Region
---
authors: 'Smith, Christine M. and Steitz, Joan A.'
title: Classification of gas5 as a Multi-Small-Nucleolar-RNA (snoRNA) Host Gene and a Member of the 5Ј-Terminal Oligopyrimidine Gene Family Reveals Common Features of snoRNA Host Genes
---
authors: 'van der Velden, Alike W. and Thomas, Adri A. M.'
title: 'The role of the 5, untranslated region of an mRNA in translation regulation during development'
---
authors: 'Yan, Jun and Marr, Thomas G.'
doi: 10.1101/gr.3109605
title: 'Computational analysis of 3Ј-ends of ESTs shows four classes of alternative polyadenylation in human, mouse, and rat'
